master/internal/api_experiment.go

```diff
@@ -966,14 +973,14 @@ func (a *apiServer) ActivateExperiment(
 func (a *apiServer) ActivateExperiments(
 	ctx context.Context, req *apiv1.ActivateExperimentsRequest,
 ) (*apiv1.ActivateExperimentsResponse, error) {
-	results, err := experiment.ActivateExperiments(ctx, req.ExperimentIds, req.Filters)
+	results, err := experiment.ActivateExperiments(ctx, req.ExperimentIds, req.Filters, req.SearchFilter)
```

Comment:
question (non-blocking): Are we expecting to deprecate req.Filters, potentially shimming it's behavior into req.SearchFilters?

Reply:
I'd prefer we come up with a more maintainable solution, req.Filters is nice because it explicitly defines it's fields & SearchFilters is nice because it gives us greater flexibility and accurately captures filters that go beyond chained ANDs.

That being said req.Filters doesn't give enough flexibility to express more complicated filters (anything beyond equality, and id exclusion) and SearchFilters is difficult to work with prior to parsing.

I had talked with @maxrussell, and a question that came up is why aren't we communicating an exhaustive list of experiment ids. My initial response had been that we don't since the frontend doesn't have the full set of experiment ids if we are working with a select all that goes across multiple pages.

Do we know how often `select all` is being used and then how often those cases exceed 1 page? I'm worried the approach we've used up to this point for bulk actions is designed for a specific edge case, rather than our user's typical usage pattern.